### PR TITLE
Time picker

### PR DESCRIPTION
**Pull request by juanfabrega**
TimePicker component. Based on the SelectInput, but generates times based on start/end and interval.
Screen Shot 2020-10-22 at 3 35 45 PM

### CODE REVIEW

File: src/components/TimePicker/TimePicker.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+   * This is a direct passthrough to the second argument of JS `toLocaleTimeString`.
+   * [More](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString)
+   */
+  dateDisplayOptions?: Intl.DateTimeFormatOptions;
+  /**
+   * End hour and minute
+   */
+  endTime?: { hour: number; minute: number; };
+  /**
+   * Interval of displayed times (in minutes). Defaults to 15 minutes.
```
Comment (nathanyoung) [inline src/components/TimePicker/TimePicker.tsx:16]: This wording is a little confusing because you set the `interval` in seconds.

**Review by nathanyoung — APPROVED** [review]
Screen Shot 2020-10-22 at 2 32 19 PM

1. Should the `options`, prop be excluded since the times are autogenerated?
2. Should there be a FormikAdapter?

**Comment by nathanyoung** [comment]
Also, might be good to have a story with the select open showing so that chromatic can capture the calculated options.

File: src/components/TimePicker/TimePicker.tsx (hunk 1 above)
Comment (juanfabrega) [reply src/components/TimePicker/TimePicker.tsx:16]: I vacilated between the two which made the docs out of sync without me realizing. Fixed now.

**Pull request merged**